libraw/structs_18.py

```diff
@@ -0,0 +1,622 @@
+""":mod:`libraw.structs` --- LibRaw struct definitions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+"""
+
+from ctypes import *  # noqa
+
+
+class ph1_t(Structure):
+
+    """Contains color data read by Phase One cameras."""
+    _fields_ = [
+        ('format', c_int),
+        ('key_off', c_int),
+        ('tag_21a', c_int),
+        ('t_black', c_int),
+        ('split_col', c_int),
+        ('black_col', c_int),
+        ('split_row', c_int),
+        ('black_row', c_int),
+        ('tag_210', c_float),
+    ]
+
+
+class libraw_iparams_t(Structure):
+
+    """The primary parameters of the image."""
+    _fields_ = [
+        ('guard', c_char * 64),
```

Comment:
Good eye; that fixes the only obvious segfault for me, but I still don't hvae time to go through and verify these structs I'm afraid. @campaul and I probably need to discuss if we're okay just merging and trusting that the structs are okay or if we really want a complete review.